package.json

```diff
@@ -25,5 +25,9 @@
   },
   "dependencies": {
     "o1js": "1.9.*"
+  },
+  "resolutions": {
```

Comment:
Does it really help, because I've tried remove the resolutions and it still work